roles/openshift_node/tasks/upgrade/restart.yml

```diff
@@ -38,12 +38,30 @@
     enabled: true
   when: openshift_use_crio | bool
 
+- name: Create ovsdb-server systemd dropin folder
+  file:
+    path: /etc/systemd/system/ovsdb-server.service.d/
+    state: directory
+  when: openshift_node_use_openshift_sdn | bool
+
+- name: Create ovsdb-server dropin config
+  ini_file:
+    path: /etc/systemd/system/ovsdb-server.service.d/timeout.conf
+    section: Service
+    option: TimeoutStartSec
+    value: 300
+  register: ovsdb_config_result
+  when: openshift_node_use_openshift_sdn | bool
+
+- name: Reload systemd units for ovsdb config change
+  command: systemctl daemon-reload
+  when: ovsdb_config_result | changed
```

Comment:
Use `is changed` instead, this is a change between Ansible 2.3 and 2.4 so release-3.7 branch should be fine.